config/config.go

```diff
@@ -390,6 +404,8 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("gdpr.usersync_if_ambiguous", false)
 	v.SetDefault("gdpr.timeouts_ms.init_vendorlist_fetches", 0)
 	v.SetDefault("gdpr.timeouts_ms.active_vendorlist_fetch", 0)
+	v.SetDefault("rates_converter.fetch_url", "http://currency.prebid.org/latest.json")
+	v.SetDefault("rates_converter.fetch_interval_seconds", 30*60) // 30 minutes
```

Comment:
Missed this one but should be 0 in a first time (desactivated), we will activated it in the last PR for this feature

Reply:
Done